week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -5,7 +5,18 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+
+//var orig = "   foo  ";
+//console.log(orig.trim()); // 'foo'
+
+function tidyUpString(strArr) {
+  var modifiedStr = " ";
+  for (i = 0; i < strArr.length; i++) {
```

Comment:
This line will work but variables are usually declared with 'let', like this:
for (let i = 0; i < strArr.length; i++) {